api/src/database/migrations/20240422A-public-registration.ts

```diff
@@ -0,0 +1,31 @@
+import type { Knex } from 'knex';
+
+// To avoid typos
```

Comment:
Bit of a nitpick, but since this migration is fairly small, I'd prefer to inline these values which would improve the readability IMO. That's also how we're doing it in the existing migrations.

Reply:
If it _strongly_ irks you or someone else, we can change it, but personally I dont mind the variables ¯\\\_(ツ)\_/¯

Reply:
Done